app/views/task_lists/index.html.erb

```diff
@@ -13,7 +13,7 @@
     <% @task_lists.each do |tasklist| %>
       <tr>
         <td><%= tasklist.name %></td>
-        <td><%= link_to 'Show', task_list_path(tasklist.id) %></td>
+        <td><%= link_to 'Show', task_list_tasks_path(tasklist.id) %></td>
         <td><%= link_to 'Edit', edit_task_list_path(tasklist) %></td>
         <td><%= link_to 'Destroy', task_list_path(tasklist.id), method: :delete, data: { confirm: 'Are you sure?' } %></td>
```

Comment:
I noticed some inconsistencies here. In some cases you supply your `.._path` with a `tasklist`. In other cases, you supply it with a `tasklist.id`. 

My suggestion is to pick one style and enforce its' consistency across the code base :)